### PR TITLE
Nyaron Race patch

### PR DESCRIPTION
**Pull request by Tostov**
## Additions

Add CE relevant stuff for Nyaron Race. Race specs themself are from the author's original patch.

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [ ] Playtested a colony (specify how long)

### CODE REVIEW

**Comment by Breadbox2k19** [comment]
UwU WuW Tostov I never knew you are the type that's into anime cat girls UwU WuW

File: Patches/Nyaron/Ammo/Ammo_Nyaron.xml [hunk 1]
```diff
@@ -0,0 +1,407 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+  <Operation Class="PatchOperationSequence">
+	<operations>
+	  <li Class="PatchOperationFindMod">
+			
+		<mods><li>Nyaron race</li></mods>
+			
+		<match Class="PatchOperationSequence">
```
Comment (N7Huntsman) [inline Patches/Nyaron/Ammo/Ammo_Nyaron.xml:9]: Double nested Patch Op Sequence?? Clean this up a bit, please. You only really need the single FindMod op with a `<match Class="PatchOperationAdd">` and add both ammo sets at once.

File: Patches/Nyaron/ThingDefs_Items/Items_Resources.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+	<Operation Class="PatchOperationSequence">
+		<success></success>
```
Comment (N7Huntsman) [inline Patches/Nyaron/ThingDefs_Items/Items_Resources.xml:4]: ?

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
Contents look fine, but the formatting needs cleaned up a bit.

**Review by N7Huntsman — APPROVED** [review]

**Pull request merged**